lms/services/lti_grading/_v13.py

```diff
@@ -67,6 +69,11 @@ def read_result(self, grading_id) -> GradingResult:
             # Due to the bug mentioned above we didn't use the API filtering by user for blackboard.
             # We'll filter ourselves here instead.
             results = [result for result in results if result["userId"] == grading_id]
+            if response.links.get("next"):
+                # We want to be notified if we are leaving results unfetched and likely start following these links
+                LOG.error(
+                    "Blackboard grading reading result, container with paginated results"
```

Comment:
Maybe the error should include the assignment ID? Or is that included contextually when the error is reported?

Reply:
Yeah, sentry should show all local variables' values